gulpfile.js

```diff
@@ -82,7 +82,7 @@ async function addExtensionPackDependencies() {
     // extension dependencies need not be installed during development
     const packageJsonContents = await fsExtra.readFile('package.json', 'utf-8');
     const packageJson = JSON.parse(packageJsonContents);
-    packageJson.extensionPack = ['ms-toolsai.jupyter', 'ms-python.vscode-pylance', 'ms-python.isort'].concat(
+    packageJson.extensionPack = ['ms-python.vscode-pylance', 'ms-python.isort'].concat(
         packageJson.extensionPack ? packageJson.extensionPack : [],
     );
```

Comment:
A solution to remove duplicates.

```suggestion
    packageJson.extensionPack = ['ms-python.vscode-pylance', 'ms-python.isort'].concat(
        packageJson.extensionPack ? packageJson.extensionPack : [],
    );
    // Remove potential duplicates.
    packageJson.extensionPack = packageJson.extensionPack.filter((item, index) => packageJson.extensionPack.indexOf(item) === index);
```